Tests.Integration.Presentation.Web/Organizations/StsOrganizationSynchronizationApiTest.cs

```diff
@@ -546,6 +546,255 @@ public async Task Can_PUT_UPDATE_Consequences_With_Conversion_Consequences()
             Assert.Null(convertedUnit.ExternalOriginUuid);
         }
 
+        [Fact]
+        public async Task Can_GET_LOGS_With_Addition_Consequences()
+        {
+            //Arrange
+            var cookie = await HttpApi.GetCookieAsync(OrganizationRole.GlobalAdmin);
+            var targetOrgUuid = await CreateOrgWithCvr(AuthorizedCvr);
```

Comment:
We are adding a lot of small unit-test-like tests here..


think we need to combine the different cases into one test, so we limit the amount of slow calls...